src/containers/email-verification/EmailVerification.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import Modal from '@mui/material/Modal'
+import IconButton from '@mui/material/IconButton'
+import CloseIcon from '@mui/icons-material/Close'
+import { styles } from './EmailVerification.styles'
+
+interface Props {
+  open: boolean,
```

Comment:
Linter is asking to delete the coma after open: boolean
```suggestion
interface Props {
  open: boolean
```

Reply:
Yep, thanks, yesterday there was a problem with linter and I couldn't figure out it. I added the suggested fixes.